homeassistant/components/plex/config_flow.py

```diff
@@ -44,12 +55,16 @@ class PlexFlowHandler(config_entries.ConfigFlow, domain=DOMAIN):
     def __init__(self):
         """Initialize the Plex flow."""
         self.current_login = {}
+        self.discovery_info = {}
         self.available_servers = None
 
     async def async_step_user(self, user_input=None):
         """Handle a flow initialized by the user."""
         if user_input is not None:
-            return await self.async_step_server_validate(user_input)
+            if user_input.pop("manual_setup", False):
+                return await self.async_step_manual_setup(user_input)
+            if CONF_TOKEN in user_input:
```

Comment:
If this isn't true, that should be an error returned in the form, right? Either token or manual should be used.